lecture2gether_flask/test.py

```diff
@@ -80,7 +79,7 @@ def test_l2go_metadata():
     meta_data_provider = L2GoMetaDataProvider('https://lecture2go.uni-hamburg.de/l2go/-/get/l/4577')
     meta_data = meta_data_provider.get_meta_data()
     assert meta_data['url'] == 'https://lecture2go.uni-hamburg.de/l2go/-/get/l/4577', f" The actual url was {meta_data['url']}"
-    assert meta_data['streamUrl'] == 'https://fms.rrz.uni-hamburg.de/vod/_definst/mp4:4l2gkoowiso/00.000_Prof.Dr.OlafAsbach_2016-03-24_14-37.mp4/playlist.m3u8', f" The actual stream url was {meta_data['streamUrl']}"
+    assert re.fullmatch(r"^https:\/\/fms\d*\.rrz\.uni-hamburg\.de\/vod\/_definst\/mp4:4l2gkoowiso\/00\.000_Prof\.Dr\.OlafAsbach_2016-03-24_14-37\.mp4\/playlist\.m3u8$", meta_data['streamUrl']) is not None, f" The actual stream url was {meta_data['streamUrl']}"
```

Comment:
```suggestion
    assert re.fullmatch(r"https://fms\d*\.rrz\.uni-hamburg\.de/vod/_definst/mp4:4l2gkoowiso/00\.000_Prof\.Dr\.OlafAsbach_2016-03-24_14-37\.mp4/playlist\.m3u8", meta_data['streamUrl']) is not None, f"The actual stream url was {meta_data['streamUrl']}"
```
With `re.fullmatch`, `^` and `$` are not needed. Also, `/` does not have to be escaped.